.github/workflows/CI-e2e-notebooks-vision.yml

```diff
@@ -5,6 +5,9 @@ on:
     branches: [main]
   pull_request:
     branches: [main]
+    paths:
+      - "responsibleai_vision/**"
```

Comment:
Do we want to add raiwidgets are well here?

https://github.com/microsoft/responsible-ai-toolbox/blob/cd2502a459925337e5020806f96c0d3d286eab91/.github/workflows/CI-notebook-text.yml#L63

Reply:
done, added raiwidgets